doc/manuals/admin/database_admin.md

```diff
@@ -97,12 +97,7 @@ the behaviour is different and the following databases are used (let
     database would be `orion-tenantA`.
 
 Per-service/tenant databases are created "on the fly" as the first
-request involving tenant data is processed by Orion. Note that there is
-a limitation in MongoDB current versions of 24,000 namespaces (each
-collection or index in a database consumes a namespace). Orion currently
-uses 5 collections per database, thus taking into account each
-collection involves also at least the `_id` index, that will end in a
-2,400 services/tenants limit (less if you have more indexes in place).
+request involving tenant data is processed by Orion.
```

Comment:
It seems that the limitation wasn't correctly interpreted. The limitation is 24,000 namespaces *for each database* not *for all databases*. Looking to https://docs.mongodb.org/manual/reference/limits/:

"For the MMAPv1 the number of namespaces is limited to the size of the namespace file divided by 628. A 16 megabyte namespace file can support approximately 24,000 namespaces. Each collection and index is a namespace."

And the namespace file (e.g. "orion.ns") is a per database artifact, i.e. it is not global for all the databases.

NTC